Composer/packages/client/src/pages/design/exportSkillModal/content/SelectDialogs.tsx

```diff
@@ -149,5 +150,10 @@ export const SelectDialogs: React.FC<ContentProps> = ({ setSelectedDialogs, proj
     []
   );
 
+  useEffect(() => {
+    for (const item of selectedDialogs) {
+      selection.setKeySelected(selection.getKey(item), true, false);
+    }
+  }, []);
```

Comment:
nit: technically you need to add selection here if it's memo, it wont make a difference but it's correct!

Reply:
use ref,